### PR TITLE
fixes #18384 - namify Guests of Host column

### PR DESCRIPTION
**Pull request by thomasmckay**
Largely a devel feature, the --prefix allows import of a csv but changing all the names to have a "prefix" specified. This is convenient for duplicating resources in-place for debug/devel.

### CODE REVIEW

File: lib/hammer_cli_csv/content_hosts.rb [hunk 1]
```diff
@@ -244,7 +244,8 @@ def update_or_create(name, line)
               },
               'subscription_facet_attributes' => {
                 'installed_products' => products(line),
-                'service_level' => line[SLA]
+                'service_level' => line[SLA],
+                'autoheal' => true
```
Comment (thomasmckay) [inline lib/hammer_cli_csv/content_hosts.rb]: inadvertent addition, remove autoheal

File: lib/hammer_cli_csv/content_hosts.rb [hunk 2]
```diff
@@ -595,6 +596,7 @@ def columns_to_csv(csv)
       def hypervisor_from_line(line)
         hypervisor = line[HOST] if !line[HOST].nil? && !line[HOST].empty?
         hypervisor ||= line[GUESTOF] if !line[GUESTOF].nil? && !line[GUESTOF].empty?
+        hypervisor = namify(hypervisor, 1) if !hypervisor.nil? && !hypervisor.empty?
```
Comment (akofink) [inline lib/hammer_cli_csv/content_hosts.rb]: I see you're following a pattern, but what's the argument against:

```rb
hypervisor = namify(hypervisor, 1) if hypervisor && !hypervisor.empty?
```

Comment (thomasmckay) [inline lib/hammer_cli_csv/content_hosts.rb]: No argument against. I will adjust the pattern in future PRs, if that is preferred.

**Comment by thomasmckay** [comment]
I did add a test for --prefix but you are correct that this prefix is global and untested. I'll open an issue to add a "devel" config to expose these hidden flags since they are useful in that mode but just not recommended for "customer" usage.

https://github.com/Katello/hammer-cli-csv/pull/152/files#diff-eb8619ccdc55ae7564320d400a48f770R410

**Comment by akofink** [comment]
Ah! I didn't see the test. Thanks for pointing that out. It got lost in all the vcr changes. :vhs:

**Pull request merged**